src/layout/Checkboxes/CheckboxesContainerComponent.tsx

```diff
@@ -110,14 +110,14 @@ export const CheckboxContainerComponent = ({
             checked={selected.includes(option.value)}
             size='small'
           >
-            {hideLabel ? null : (
-              <span className={classes.checkBoxLabelContainer}>
+            {
+              <span className={`${hideLabel ? 'sr-only' : ''} ${classes.checkBoxLabelContainer}`}>
```

Comment:
Nitpick, but I think its cleaner to use `classnames` package to combine css classes, instead of template strings with possibly empty parts